armi/runLog.py

```diff
@@ -656,10 +644,18 @@ def warningReport(self):
             return
 
         # sort by labcollections.defaultdict(lambda: 1)
+        total = 0
         for label, count in sorted(
-            dupsFilter.singleWarningMessageCounts.items(), key=operator.itemgetter(1)
+            dupsFilter.warningCounts.items(), key=operator.itemgetter(1), reverse=True
         ):
-            self.info("  {0:^10s}   {1:^25s}".format(str(count), str(label)))
+            self.info(f"  {str(count):^10s}   {str(label):^25s}")
+            total += count
+        self.info("------------------------------------")
+
+        # add a totals line
+        self.info(
+            "  {0:^10s}   {1:^25s}".format(str(total), str("Total Number of Warnings"))
```

Comment:
no f-strings here?

Reply:
I used `ruff` to fix these f-strings. And this one `ruff` couldn't fix.

Which is a strange little `ruff` quirk. I'm not sure why.

I guess I should open a `ruff` ticket!

Reply:
Okay, I opened a ticket: https://github.com/astral-sh/ruff/issues/17972

Reply:
You can manually f-string it or no? 

suggest (if it works): 
`f"  {str(total):10s}   Total Number of Warnings"`